yarn.lock

```diff
@@ -2605,14 +2605,6 @@
     slice-ansi "^4.0.0"
     string-width "^4.2.0"
 
-"@parcel/codeframe@^2.0.0-alpha.3":
```

Comment:
no updates to package.json's, why is this getting updated?
I'm ok if you refreshed it on purpose

Reply:
I guess it updated when I ran yarn. Perhaps the one on master is out of date?